.github/workflows/docker.yml

```diff
@@ -17,6 +17,9 @@ on:
   workflow_dispatch:
   release:
     types: [published, edited]
+    permissions:

```

Comment:
It's a top-level key, it doesnt go inside this section

Reply:
My apologies, you're absolutely right! I've corrected it now with a force push.